src/main/java/com/zenjava/javafx/maven/plugin/AbstractJfxToolsMojo.java

```diff
@@ -73,7 +73,7 @@
      * <p>
      * This defaults to 'target/jfx/app' and in most cases there is no real need to change this.
      *
-     * @parameter property="jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
+     * @parameter property="jfx.jfxAppOutputDir" default-value="${project.build.directory}/jfx/app"
```

Comment:
Currently I just override some defaults and bind the JavaFX jar goal to the package phase:
https://github.com/Drombler/drombler-fx/blob/137-packaging-support/drombler-fx-maven-plugin/src/main/resources/META-INF/plexus/components.xml

This allows developers to configure the JavaFX Maven Plugin (e.g. in the pluginManagement section) as they are used to in non-Drombler FX application.

I'm not sure this would be the case if used the Maven Invoker Plugin to call the JavaFX Maven Plugin.